include/devices/esp32.h

```diff
@@ -24,13 +24,18 @@ class NativeESP32 : public OswTemperatureProvider, public OswTimeProvider {
         return 20;
     }; // This sensor is not sooo good...
 
+    void setClockResyncEnabled(const bool& enable);
+    bool isClockResyncEnabled();
```

Comment:
`setClockResyncEnabled` is used - the `isClockResyncEnabled` not. Regardless, both functions were added in case someone wants to control that behavior down the road. For now it is only used by the ESP32 itself - but for the future users can modify their sources to disable it at all.